yb-voyager/src/srcdb/yugabytedb.go

```diff
@@ -391,7 +392,10 @@ func (yb *YugabyteDB) GetColumnToSequenceMap(tableList []*sqlname.SourceName) ma
 
 func (yb *YugabyteDB) GetServers() string {
```

Comment:
This code has become confusing. It returns all nodes of the cluster -- but with master port! The port will not be valid for all nodes.

It will be better to return a list of ipOrHostNames of all the servers from this method (that will also justify the name). The caller is then free to append each ipOrHostName with any desired port before using it.